docs/user-guide/day2-master/README.md

```diff
@@ -0,0 +1,20 @@
+# Installing control plane (master node) as day 2 operation
```

Comment:
The majority of the document focuses on the cluster itself, rather than on Assisted.  Is there any difference between Assisted SaaS and MCE/ACM in this regard?  Is it just how to download the ISO for the master?

Reply:
This is right, that's because the whole process is really an OCP process and not Assisted's feature. On the side of Assisted Service the only difference is how you get an ISO for day2. I intentionally don't describe this here as we have enough documentation about day2 process itself.

> Is there any difference between Assisted SaaS and MCE/ACM in this regard

The difference is more about whether you do ZTP or BIY, not SaaS vs ACM